lib/diagnostic.js

```diff
@@ -24,7 +24,12 @@ const sum = function sum() {
 // if invoking the callback for any element
 // in the array returns a truthy value
 const any = function any(array, callback) {
-
+  for (let i = 0; i < array.length; i++)  {
+    if (callback(array[i], i, array) === true)  {
```

Comment:
It's not necessary to use ` === true` here; if the callback is returning a truthy value, that's enough to tell the `if` which way to go.